ApplicationLibCode/ProjectDataModel/Summary/RimEnsembleCurveSet.h

```diff
@@ -180,7 +180,9 @@ class RimEnsembleCurveSet : public caf::PdmObject, public RimEnsembleCurveSetInt
     std::vector<cvf::Color3f> generateColorsForCases( const std::vector<RimSummaryCase*>& summaryCases ) const;
 
     RiuPlotAxis axisY() const;
+    RiuPlotAxis axisX() const;
     void        setLeftOrRightAxisY( RiuPlotAxis plotAxis );
+    void        setBottomOrTopAxis( RiuPlotAxis plotAxis );
```

Comment:
The two names are not "symmetric": either remove the Y in `setLeftOrRightY`, or add X in `setBottomOrTopAxis`

Reply:
Yes, harmonized with RimSummaryCurve